src/app/app.routes.ts

```diff
@@ -1,3 +1,65 @@
 import { Routes } from '@angular/router';
 
-export const routes: Routes = [];
+export const routes: Routes = [
+  {
+    path: 'about',
+    loadComponent: () =>
+      import('./pages/about/about.component').then((m) => m.AboutComponent),
+  },
+  {
+    path: 'auth',
+    loadComponent: () =>
+      import('./pages/auth/auth.component').then((m) => m.AuthComponent),
+    children: [
+      {
+        path: 'login',
+        loadComponent: () =>
+          import('./pages/auth/login/login.component').then(
+            (m) => m.LoginComponent
+          ),
+      },
+      {
+        path: 'signup',
+        loadComponent: () =>
+          import('./pages/auth/signup/signup.component').then(
+            (m) => m.SignupComponent
+          ),
+      },
+    ],
+  },
+  {
+    path: 'dashboard',
+    loadComponent: () =>
+      import('./pages/dashboard/dashboard.component').then(
+        (m) => m.DashboardComponent
+      ),
+    children: [
```

Comment:
This shouldn't be a child of `dashboard` route, but a sibling

Reply:
you mean, it should be an independent route?:
{
path: 'dashboard/projects',

loadComponent: () =>
          import('./pages/dashboard/projects/projects.component').then(
            (m) => m.ProjectsComponent
          ),
}